docs/src/AvatarGroup.doc.js

```diff
@@ -0,0 +1,505 @@
+// @flow strict
+import React, { type Node } from 'react';
+import { AvatarGroup, Box } from 'gestalt';
+import PropTable from './components/PropTable.js';
+import PageHeader from './components/PageHeader.js';
+import MainSection from './components/MainSection.js';
+import FeedbackCallout from './components/FeedbackCallout.js';
+import CombinationNew from './components/CombinationNew.js';
+
+const cards: Array<Node> = [];
+const card = (c) => cards.push(c);
+
+card(<FeedbackCallout componentName="AvatarGroup" />);
```

Comment:
What does this mean?

Reply:
We have to add new components to the available components on the google form, otherwise the correct component won't be auto-selected (or available to select at all). I'll Slack you the link

Reply:
DONE